aws/resource_aws_ses_domain_mail_from.go

```diff
@@ -0,0 +1,96 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesDomainMailFrom() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesDomainMailFromCreate,
+		Read:   resourceAwsSesDomainMailFromRead,
+		Delete: resourceAwsSesDomainMailFromDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"domain": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"mail_from_domain": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceAwsSesDomainMailFromCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).sesConn
+
+	domainName := d.Get("domain").(string)
+	mailFromDomain := d.Get("mail_from_domain").(string)
+
+	createOpts := &ses.SetIdentityMailFromDomainInput{
+		BehaviorOnMXFailure: aws.String("UseDefaultValue"),
```

Comment:
Two things here:
* Nitpick: SDK has a constant available `ses.BehaviorOnMXFailureUseDefaultValue`
* Seems easy enough to support this as an optional attribute, that defaults to this